applications/StructuralMechanicsApplication/custom_constitutive/small_strain_isotropic_damage_3d.cpp

```diff
@@ -248,7 +178,8 @@ void SmallStrainIsotropicDamage3D::CalculateStressResponse(
             const double damage_rate = (stress_variable - hardening_modulus * rStrainVariable)
                                        / (rStrainVariable * rStrainVariable * rStrainVariable);
             r_constitutive_matrix *= (1. - damage_variable);
-            r_constitutive_matrix -= damage_rate * outer_prod(r_stress_vector, r_stress_vector);
+            r_constitutive_matrix -= damage_rate * outer_prod(r_stress_vector, r_stress_vector_pos);
+            // real stress = (1-d) * effective stress
```

Comment:
Remove if not necessary

Reply:
It's a comment. Someone mentioned that the line below wasn't clear from the physical point of view

Reply:
OK